metadata-ingestion-modules/airflow-plugin/tests/integration/test_plugin.py

```diff
@@ -132,6 +136,7 @@ def _run_airflow(
         # Point airflow to the DAGs folder.
         "AIRFLOW__CORE__LOAD_EXAMPLES": "False",
         "AIRFLOW__CORE__DAGS_FOLDER": str(dags_folder),
+        "AIRFLOW__CORE__PLUGINS_FOLDER": str(plugins_folder),
```

Comment:
once you change the code to define the CustomOperator in the same file as the dag, this won't be necessary